chain/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! This Library contains the `Chain Service` implement:
 //!
-//! - [Chain](chain::chain::Chain) represent a struct which
+//! - [ChainService](chain::chain::ChainService) represent a struct which implement `ChainService`
```

Comment:
`xx:yy` is stabilized in Rust 1.38.0, it's better to use `chain/struct.ChainController.html` instead.

The module doc is confusing: `X is xx which implements X`. Let's just change the module doc to

```
//! CKB chain service.
```